enterprise/frontend/src/embedding-sdk/conventional-changelog-config.js

```diff
@@ -1,4 +1,51 @@
 // eslint-disable-next-line import/no-commonjs
 module.exports = {
-  gitRawCommitsOpts: { grep: "(sdk)" },
+  gitRawCommitsOpts: {
+    grep: "(sdk)",
+  },
+  options: {
+    preset: "angular",
+    pkg: {
+      path: "enterprise/frontend/src/embedding-sdk/package.template.json",
+    },
+    tagPrefix: "embedding-sdk-",
+    transform: function (commit, cb) {
```

Comment:
The function is based on the default function from `conventional-changelog-core`

https://github.com/conventional-changelog/conventional-changelog/blob/65550d9a30cfab4b00b3e982e42bd68fedafcefc/packages/conventional-changelog-core/lib/merge-config.js#L83-L98